src/plugins/intel_gpu/tests/functional/single_layer_tests/dynamic/convolution.cpp

```diff
@@ -88,6 +88,13 @@ class ConvolutionLayerGPUTestDynamic : public testing::WithParamInterface<convLa
         size_t convOutChannels;
         std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType) = convParams;
 
+        // WA: check data when input shape is dynamic and pad is exist.
+        //     If 1d conv, 1d pad should be applied to y axis. But there was a bug what it applied to x axis.
+        if (inputShape.first.is_dynamic() && padBegin.size() == 1 && padBegin[0] == 1 && padEnd.size() == 1 && padEnd[0] == 1) {
+            abs_threshold = 9;
```

Comment:
Why? That doesn't look like a reasonable threshold

Reply:
At almost cases, they do not care about the compare results. There is no parameter to set those threshold values. I chose that threshold values after checking that there was a lot of difference in value when the pad was calculated incorrectly.

Reply:
The threshold condition is not very intuitive. We will revisit it after merge.